activities/communication/community-call.md

```diff
@@ -0,0 +1,19 @@
+---
+type: Resource
+---
+
+# Foundation for Public Code community call
+
+We run regular [community calls](../community-calls/index.md) for the Foundation for Public Code.
+
+## Preparing
+
+You can look at the preparations for the [Standard for Public Code community call](../standard-maintainance/preparing-community-call.md) and base your preparations off that.
+
+## Running
+
+You can follow the steps in the guide for [running community call](running-community-call.md).
+
+## Following up
+
+* [Post process the community](../community-calls/post-process-community-call.md) call including updating issues and pull requests on the [About repository](https://github.com/publiccodenet/about)
```

Comment:
moved a word so the hyperlink text makes sense on its own to someone navigating using a screenreader.

```suggestion
* [Post process the community call](../community-calls/post-process-community-call.md), including updating issues and pull requests on the [About repository](https://github.com/publiccodenet/about).
```